src/views/send/index.js

```diff
@@ -234,113 +234,39 @@ class Send extends Component {
                 });
                 return;
             }
-            // Update the USD value element
-            const usdAmountElement = document.getElementById("pokt-amount-usd");
-            const usdAmountElementText = document.getElementById("modal-usd-amount-to-send");
             
-            // Check
-            if (usdAmountElement && usdAmountElementText) {
-                // Convert the decimals to upokt to use the value for the send-tx
-                const value = amountElement.value * 1000000;
-                // Convert pokt to usd
-                const usdValue = Number(Config.POKT_USD_VALUE) * amountElement.value;
-                
-                // Check if the amount to send doesn't exceed the current balance
-                const {upoktBalance, amountToSend} = this.state;
-
-                if (upoktBalance < amountToSend) {
-                    // Update the state
-                    this.setState({
-                        amountToSend: value,
-                        poktAmount: amountElement.value,
-                        poktAmountUsd: usdValue.toFixed(2),
-                        modalAmountToSendPokt: `${amountElement.value} POKT`,
-                        modalAmountToSendUsd: `${usdValue.toFixed(2)} USD`,
-                        isAmountValid: false,
-                        amountError: "Insufficient balance."
-                    });
-                    // Disable loader indicator
-                    this.enableLoaderIndicatory(false);
-                } else {
-                    // Save the values in the state
-                    this.setState({
-                        amountToSend: value,
-                        isAmountValid: true,
-                        poktAmount: amountElement.value,
-                        poktAmountUsd: usdValue.toFixed(2),
-                        modalAmountToSendPokt: `${amountElement.value} POKT`,
-                        modalAmountToSendUsd: `${usdValue.toFixed(2)} USD`
-                    });
-                    
-                    // Remove error message
-                    this.toggleAmountError(undefined);
-                }
-            }
-        } else {
-            this.toggleAmountError("Amount is invalid.");
-            this.setState({isAmountValid: false});
-            return;
-        }
-    }
+            // Convert the decimals to upokt to use the value for the send-tx
+            // Math.round to remove the decimals
+            const upoktValue = Math.round(amountValue * 1000000);
```

Comment:
This might probably overflow, but considering we're soon revamping the wallet, we can probably let this one slip by...